frontend/packages/client/src/components/Community/CommunityEditorDetails.js

```diff
@@ -10,6 +10,15 @@ import useFlowAddrValidator, {
 } from './hooks/useFlowAddrValidator';
 import FadeIn from 'components/FadeIn';
 
+const determineAutoFocus = (listLength, index, addr, autoFocusOnLoad) => {
+  return (
+    // fist element of list depends on configuration (autoFocusOnLoad)
+    (listLength === 1 && addr === '' && autoFocusOnLoad) ||
+    // new elements on list will be autofocus by default
+    (listLength === index + 1 && addr === '' && index !== 0)
+  );
+};
+
```

Comment:
Any way we could simplify this by using ref? Or does the current component structure prevent us from being able to take this approach?
```
useEffect(() => {
        mainRef.current.focus();
    }, [mainRef]);

<
 ```